### DISCUSSION TITLE
Missing link to example input file

### DISCUSSION THREAD

**Discussion by libinbabu1991** [idaholab/TMAP8#187] [General]
Hi, 

I woud like to try the Divertor monoblock example. in TMAP8. I think the link to the input file is missing at the end of the page. 

https://github.com/idaholab/TMAP8/blob/devel/doc/content/examples/divertor_monoblock/index.md

I appreciate if I someone could update the link and provide the complete example package including the mesh file.

Thanks!

**Comment by simopier** [comment]
Hi @libinbabu1991, please check the documentation website, with the working link (which also shows the path to the input file once you download TMAP8):
https://mooseframework.inl.gov/TMAP8/examples/divertor_monoblock/index.html

The complete package is available here: 
https://mooseframework.inl.gov/TMAP8/getting_started/installation.html

**Reply by simopier** [reply]
If you already installed TMAP8, then do `cd ~/projects/TMAP8[test/tests/divertor_monoblock/` and you'll see everything you need. Note that the mesh is automatically generated within the input file.

**Reply by libinbabu1991** [reply]
Thanks for your message!

I tried copying the code given in https://mooseframework.inl.gov/TMAP8/examples/divertor_monoblock/index.html into a notepad file and use as the input file. It gave me a warning :  Check to make sure that it exists and that you have read permission.

However, I found a working input file from the link you provided me (cd ~/projects/TMAP8[test/tests/divertor_monoblock/)  and I am able to run it now!

Thank you very much!
